### PR TITLE
Added information about logging SQL queries

### LINKED ISSUE #1903: Add a note about how to log database queries to the LOCAL_DEVELOPMENT.md
Instructions are in #1896

### PR DESCRIPTION
**Pull request by daveverwer**
Fixes #1903

### CODE REVIEW

File: LOCAL_DEVELOPMENT_SETUP.md [hunk 1]
```diff
@@ -60,13 +60,29 @@ make serve-front-end
 
 Navigate to `http://127.0.0.1:8080` with a web browser, and the site should be up and running locally with CSS and JavaScript working!
 
+## Logging database queries
+
+It can be helpful when working with code that makes database queries to see the SQL statements that get executed. You can switch on logging of _all_ executed SQL statements by uncommenting this line in `configure.swift`:
+
+```swift
+app.logger.logLevel = .debug
```
Comment (finestructure) [inline LOCAL_DEVELOPMENT_SETUP.md:68]: This isn't quite what I'd intended with the original change. Let me recap that.

The default log level is `info` and the default sql logging is on `debug`. With the line that I added in `configure.swift`:

```swift
                                // Set this to .info to log SQL queries with the default log level.
                                sqlLogLevel: .debug),
```

I'm merely re-stating the default.

The instruction for users then is to change this to `.info`, in order to enable SQL logging on the default logging level of `info`.

The advantage here is that you don't need to lower the log level to `debug` and get all sort of other debug logging alongside the SQL that you're probably not interested in at that point.

File: LOCAL_DEVELOPMENT_SETUP.md [hunk 2]
```diff
@@ -60,13 +60,29 @@ make serve-front-end
...
+```
```
Comment (finestructure) [inline LOCAL_DEVELOPMENT_SETUP.md:69]: This isn't quite what I'd intended with the original change. Let me recap that.

The default log level is `info` and the default sql logging is on `debug`. With the line that I added in `configure.swift`:

```swift
                                // Set this to .info to log SQL queries with the default log level.
                                sqlLogLevel: .debug),
```

I'm merely re-stating the default.

The instruction for users then is to change this to `.info`, in order to enable SQL logging on the default logging level of `info`.

The advantage here is that you don't need to lower the log level to `debug` and get all sort of other debug logging alongside the SQL that you're probably not interested in at that point.

Comment (daveverwer) [reply LOCAL_DEVELOPMENT_SETUP.md]: Fixed

File: LOCAL_DEVELOPMENT_SETUP.md [hunk 3]
```diff
@@ -60,13 +60,25 @@ make serve-front-end
...
... "repositories"."last_pull_request_closed_at" AS "repositories_last_pull_request_closed_at", "repositories"."license" AS "repositories_license", "repositories"."license_url" AS "repositories_license_url", "repositories"."name" AS "repositories_name", "repositories"."open_issues" AS "repositories_open_issues", "repositories"."open_pull_requests" AS "repositories_open_pull_requests", "repositories"."owner" AS "repositories_owner", "repositories"."owner_name" AS "repositories_owner_name", "repositories"."owner_avatar_url" AS "repositories_owner_avatar_url", "repositories"."readme_url" AS "repositories_readme_url", "repositories"."readme_html_url" AS "repositories_readme_html_url", "repositories"."releases" AS "repositories_releases", "repositories"."stars" AS "repositories_stars", "repositories"."summary" AS "repositories_summary" FROM "packages" LEFT JOIN "repositories" ON "packages"."id" = "repositories"."package_id" WHERE "repositories"."owner" ilike $1 AND "repositories"."name" ilike $2 LIMIT 1 [Optional("daveverwer"), Optional("LeftPad")] [component: server, database-id: psql, request-id: 56DF427B-1B2D-46FD-9AED-68CDE3C52B7C] (FluentPostgresDriver/FluentPostgresDatabase.swift:29)
```
Comment (finestructure) [inline LOCAL_DEVELOPMENT_SETUP.md:70]:
```suggestion
[ INFO ] SELECT "packages"."id" AS "packages_id", "packages"."created_at" AS "packages_created_at", "packages"."updated_at" AS "packages_updated_at", "packages"."platform_compatibility" AS "packages_platform_compatibility", "packages"."processing_stage" AS "packages_processing_stage", "packages"."score" AS "packages_score", "packages"."status" AS "packages_status", "packages"."url" AS "packages_url", "repositories"."id" AS "repositories_id", "repositories"."created_at" AS "repositories_created_at", "repositories"."updated_at" AS "repositories_updated_at", "repositories"."forked_from_id" AS "repositories_forked_from_id", "repositories"."package_id" AS "repositories_package_id", "repositories"."authors" AS "repositories_authors", "repositories"."commit_count" AS "repositories_commit_count", "repositories"."default_branch" AS "repositories_default_branch", "repositories"."first_commit_date" AS "repositories_first_commit_date", "repositories"."forks" AS "repositories_forks", "repositories"."homepage_url" AS "repositories_homepage_url", "repositories"."is_archived" AS "repositories_is_archived", "repositories"."is_in_organization" AS "repositories_is_in_organization", "repositories"."keywords" AS "repositories_keywords", "repositories"."last_activity_at" AS "repositories_last_activity_at", "repositories"."last_commit_date" AS "repositories_last_commit_date", "repositories"."last_issue_closed_at" AS "repositories_last_issue_closed_at", "repositories"."last_pull_request_closed_at" AS "repositories_last_pull_request_closed_at", "repositories"."license" AS "repositories_license", "repositories"."license_url" AS "repositories_license_url", "repositories"."name" AS "repositories_name", "repositories"."open_issues" AS "repositories_open_issues", "repositories"."open_pull_requests" AS "repositories_open_pull_requests", "repositories"."owner" AS "repositories_owner", "repositories"."owner_name" AS "repositories_owner_name", "repositories"."owner_avatar_url" AS "repositories_owner_avatar_url", "repositories"."readme_url" AS "repositories_readme_url", "repositories"."readme_html_url" AS "repositories_readme_html_url", "repositories"."releases" AS "repositories_releases", "repositories"."stars" AS "repositories_stars", "repositories"."summary" AS "repositories_summary" FROM "packages" LEFT JOIN "repositories" ON "packages"."id" = "repositories"."package_id" WHERE "repositories"."owner" ilike $1 AND "repositories"."name" ilike $2 LIMIT 1 [Optional("daveverwer"), Optional("LeftPad")] [component: server, database-id: psql, request-id: 56DF427B-1B2D-46FD-9AED-68CDE3C52B7C] (FluentPostgresDriver/FluentPostgresDatabase.swift:29)
```

:)

Comment (daveverwer) [reply LOCAL_DEVELOPMENT_SETUP.md]: Attention to detail ++ 😂

**Comment by samdeane** [comment]
Is there any other value that’s ever going to be useful to pass in here, other than the default (`.debug`), or the one to enable logging (`.info`)?

If not, it might be clearer to just have a boolean parameter `logSQLQueries`.

The fact that one has to move “down” in detail level from `.debug` to `.info` in order to obtain more information is counter-intuitive.

**Pull request merged**